template/src/pages/home/home.tsx

```diff
@@ -1,23 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import globalStyles from 'assets/stylesheets/global-styles.module.scss';
 import { AppLink } from 'routes/app-link';
 import { RouteName } from 'routes/routes';
-
 import { Button } from 'common/button';
 import { Avatar } from 'common/avatar';
+import { Modal } from 'common/modal';
+import { ModalSizes } from 'common/modal/modal';
 import styles from './home.module.scss';
 
-const Home = () => (
-  <div className={globalStyles.genericContainer}>
-    <h1 className={styles.title}>
-      <Avatar size="xl" />
-      <hr />
-      Welcome! This is the homepage.
-      <AppLink routeName={RouteName.About}>About</AppLink>
-      <Button>Hola Mundo</Button>
-    </h1>
-  </div>
-);
+const Home = () => {
+  const [openModal, setOpenModal] = useState(false);
+  const closeModal: () => void = () => { setOpenModal(false); };
+  const position = styles.modalPosition;
+  return (
+    <div className={globalStyles.genericContainer}>
+      <h1 className={styles.title}>
+        <Avatar size="xl" />
+        <hr />
+        Welcome! This is the homepage.
+        <AppLink routeName={RouteName.About}>About</AppLink>
+        <Button onClick={() => setOpenModal(true)}>Hola Mundo</Button>
+      </h1>
+      {openModal && (
+        <Modal
+          isOpen={openModal}
+          onClose={closeModal}
+          size={ModalSizes.small}
+          modalClassName={position}
+          ariaLabelledby="modal-example"
```

Comment:
This does not point to anything, right? It should reference a valid element on the DOM